pytorch_lightning/loggers/neptune.py

```diff
@@ -209,7 +209,7 @@ def __init__(self,
         self.tags = tags
         self._kwargs = kwargs
         self._experiment_id = None
-        self._experiment = self._create_or_get_experiment()
+        self._experiment = None
```

Comment:
```suggestion
        self._experiment = self.experiment
```
try this, this will only run on rank 0

Reply:
That won't work because `self._experiment` does not exist at that point.

Reply:
oh yes, silly me
ok, need to check tests why the experiment is required then. 
with your change, we need to fix the tests somehow